### ISSUE TITLE
Not really sure what went wrong, but VSCode told me to post something here.

### ISSUE THREAD

**Issue by Rwelk** [PowerShell/PSReadLine#1887]
I was working on a school project when I was suddenly hit with an error when trying to compile. I think it's a problem with either VSCode or Pylance? I'll be honest, I have no clue what went wrong, how I broke it, or even where I am, I was just told to post this here.

```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new      
-----------------------------------------------------------------------Last 118 Keys:
 & Space " C : / P r o g r a m Space F i l e s / P y t h o n 3 8 / p y 
t h o n . e x e " Space " c : / U s e r s / m a g i c / O n e D r i v e / C l a s s Space N o t e s / C S Space S e n i o r Space P r o j e c 
t / C o n s o l e Space Q u e s t / m a p . p y " Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```

**Comment by OLedelay** [comment]
I had roughly the same with an actual value of 15, might this be related to #1306 with VSCode using an older version?
```
Actual value was 15.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
```

**Comment by OLedelay** [comment]
You're also supposed to follow the issue template I believe:

> The maintainer may close your issue without further explanation or engagement if:
> - You delete this entire template and go your own path;
> - You file an issue that has many duplicates;
> - You file an issue completely blank in the body.

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0-rc1 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0-rc1) from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
